src/app/utils/field-validation/config.js

```diff
@@ -2,15 +2,6 @@ const FIELDS_TO_REJECT = require('../../../shared/resources/basic')
   .types.filter((f) => !f.submitted)
   .map((f) => f.name)
 
-// deprecated
-const ALLOWED_VALIDATORS = [
-  'TableValidator',
-  // BaseFieldValidator can be constructed by the FieldValidatorFactory,
-  // but is missing from this list.
-  // 'BaseFieldValidator',
-]
-
 module.exports = {
   FIELDS_TO_REJECT,
```

Comment:
seems like `FIELDS_TO_REJECT` is the only variable left here. could you help to see if we're able to remove this file entirely?

Reply:
shifted to file with the field types